people_tool/templates/people_tool/index.html

```diff
@@ -53,7 +52,6 @@
     <script src="{% static 'people_tool/js/app.js' %}"></script>
     <script src="{% static 'people_tool/js/controllers/SearchController.js' %}"></script>
     <script src="{% static 'people_tool/js/controllers/CoursesController.js' %}"></script>
-    <script src="{% static 'people_tool/js/directives/BadgeDirective.js' %}"></script>
```

Comment:
Would be good to cleanup/delete this file as well, in order to avoid confusion in the future.